ts/parts-more/PackedBubbleSeries.ts

```diff
@@ -1447,6 +1451,79 @@ seriesType<Highcharts.PackedBubbleSeries>(
                 nextBubble[4]
             ]; // the same as described before
         },
+
+        /**
+         * Get non-presentational attributes for a point. Used internally for
+         * both styled mode and classic. Can be overridden for different series
+         * types.
+         *
+         * @see Series#pointAttribs
+         *
+         * @function Highcharts.Series#markerAttribs
+         *
+         * @param {Highcharts.Point} point
+         *        The Point to inspect.
+         *
+         * @param {string} [state]
+         *        The state, can be either `hover`, `select` or undefined.
+         *
+         * @return {Highcharts.SVGAttributes}
+         *         A hash containing those attributes that are not settable from
+         *         CSS.
+         */
+        markerAttribs: function (
```

Comment:
Is all this just to fix the problem with `Math.round`? In that case we should just have a series property that dictates whether the position should be rounded...

Reply:
Definitely I am with you on this one! I was afraid to make this kind of adjustment in Highcharts core but I will do it in my next commit.

Reply:
Edited - used crisp option as discussed